omplbindings/bindings.go

```diff
@@ -17,33 +17,23 @@ package main
 */
```

Comment:
Can we make a new repo, move all of the C++ and OMPL-related stuff there, and this repo can be purely for testing RDK?

It doesn't really make sense to store our internal testing tools in a folder called "omplbindings".

Reply:
It doesnt but I think this is something we should live with for now and fix later.  In general there is a lot to be cleaned up on this repo that is now public, so if we care about it we should ticket out a refactor/cleanup.

Regarding keeping this in here vs moving to RDK.  I thought about putting it into RDK a few times but I think having it live here makes sense (at least for the time being) because we can use the common scenes to test against other algorithms (like ompl if we go back that direction) without needing to export them from RDK (which doesn't really make sense to me)

Reply:
I'm not suggesting testing things be removed from this repo and put in RDK- this is the `motion-testing` repo after all.

I'm suggesting things that aren't motion tenting (everything OMPL) be removed from this repo.

Reply:
Basically I'm suggesting:
1) make new (private for now) repo for OMPL-related things
2) duplicate current contents of this repo to that
3) delete everything in this repo that isn't actually needed/used for motion CI

Reply:
Oh.  Yeah I'm on board with this

Reply:
https://github.com/viamrobotics/ompl-bridge now exists for all your ompl-related needs

Reply:
Yeah I'm fine with this

Reply:
Should probably change the overarching folder name, too